mlir/include/mlir/Dialect/LLVMIR/LLVMOpBase.td

```diff
@@ -139,6 +139,16 @@ class LLVM_VectorOf<Type element> : Type<
 class LLVM_ScalarOrVectorOf<Type element> :
     AnyTypeOf<[element, LLVM_VectorOf<element>]>;
 
+// Type constraint accepting an LLVM fixed vector type with an additional constraint
+// on the vector element type.
+class LLVM_FixedVectorOf<Type element> : Type<
+  And<[LLVM_AnyFixedVector.predicate,
```

Comment:
I think you can use `LLVM_AnyFixedVector` : https://github.com/llvm/llvm-project/blob/f3c408d1726f6a921212faf68085f68bf8533f0c/mlir/include/mlir/Dialect/LLVMIR/LLVMOpBase.td#L119

Reply:
according to documentation https://llvm.org/docs/LangRef.html#i-store
"fixed vector of floating-point type"

Reply:
What I was pointing to was that you can use:
```
def LLVM_AtomicRMWType
    : AnyTypeOf<[LLVM_AnyFloat, LLVM_AnyPointer, AnySignlessInteger, LLVM_AnyFixedVector]>;
```
And then check for fp16 in the verifier. In this way you don't have to add `LLVM_FixedVectorOf`.